src/components/Local.vue

```diff
@@ -61,6 +61,7 @@
           </v-btn>
         </span>
         <span>{{ clip.title }}</span>
+        <span class="small-info"> Created on: {{ clip.created_on }} </span>
```

Comment:
should we add this feature, where the date field is a link and onclick, we show all the clips generated on that date ?

Reply:
"Filtering by Date" is a good thing IMO. What if we implement it in the filter panel (which is yet to be developed). Let me know what you think.

Reply:
I had never given thought to filter panel, I'd add this to a TODO in project, Also

I think that there is also discussions panel in Github now, so I would try to use that if possible from now on.